bats/fb-test-backup.bats

```diff
@@ -58,6 +58,18 @@ BACKUP_ONLINE_DIR="${BACKUP_BASE_DIR}/online"
   tFileExists "${BACKUP_OFFLINE_DIR}/pulp_data.tar"
 }
 
+@test "check offline backup contents for Foreman Proxy" {
```

Comment:
This test only works on content proxies. I'm not entirely sure if this is the right way:
```suggestion
@test "check offline backup contents for Foreman Proxy Content" {
```

Then it should also test for `foreman-installer-katello` to be present or otherwise skip it.